versions.gradle

```diff
@@ -0,0 +1,117 @@
+ext.deps = [:]
```

Comment:
add a comment at the beginning of the file explaining why you put all the version here.

Reply:
ack